### ISSUE TITLE
Empty box on Windows

### ISSUE THREAD

**Issue by pbankovic** [dorkbox/Notify#4]
Doesn't show any text on Windows. Only empty box.

**Comment by pbankovic** [comment]
Additional info:

Call:
public static void main(String[] args) {
		Notify.create().title("TEST TITLE").text("TEST TEXT").darkStyle().hideAfter(3000).setScreen(1).show();
		
}

Result:
![notify](https://cloud.githubusercontent.com/assets/24352009/20858808/cefd920c-b94d-11e6-802a-e0c394eed04b.jpg)

**Comment by dorkbox** [comment]
What is the java version, and what is the windows version?

**Comment by pbankovic** [comment]
java version "1.7.0_79"
Java(TM) SE Runtime Environment (build 1.7.0_79-b15)
Java HotSpot(TM) Client VM (build 24.79-b02, mixed mode, sharing)

Microsoft Windows [Version 10.0.14393]

**Comment by dorkbox** [comment]
This one was really tough. An update to java broke behavior... It's now fixed in v2.19 via bbdb394cd2f2ee5a385f3a91463640bd32d05166

**Comment by MaikoID** [comment]
Hi I'm getting the same problem when stressing it out send various notifications. A empty notification overlaps a normal one and I need to click on it to get rid of it.

Is easy to reproduce sending more than five notifications.

Ubuntu 16.10; JAVA 1.7*
more easy reproductible if I change the Java version to 1.8

`		notify = Notify.create().title(title).text(args).hideAfter(3000).position(Pos.TOP_RIGHT).darkStyle()
				.onAction(new ActionHandler<Notify>() {
					@Override
					public void handle(final Notify arg0) {
					}
				}).graphic(icon);
		notify.show()`

**Comment by dorkbox** [comment]
What version is this with? Can you double check that it is still happening for `v2.20`?

**Comment by MaikoID** [comment]
yeap
```
		<dependency>
			<groupId>com.dorkbox</groupId>
			<artifactId>Notify</artifactId>
			<version>2.20</version>
		</dependency>
```

**Comment by dorkbox** [comment]
ok. i'll figure out what's going on.

**Comment by dorkbox** [comment]
Fixed in commit f8866c9aeb25af842bc75f34410495201cd4267b
